### ISSUE TITLE
quadprog usage

### ISSUE THREAD

**Issue by smshariatzadeh** [HiroIshida/robust-tube-mpc#12]
Thanks for sharing your nice code.

Could you please explain why you use the quadprog command in a loop in [OptimalController.m](   https://github.com/HiroIshida/robust-tube-mpc/blob/b82f0c40eed287eaafac097aa5990ded35b301b5/src/OptimalControler.m#L75  )) and what is the role of the following command? ([in optimalcontroler.m line 79](https://github.com/HiroIshida/robust-tube-mpc/blob/b82f0c40eed287eaafac097aa5990ded35b301b5/src/OptimalControler.m#L79)):

 C_ineq1_relaxed = C_ineq1_relaxed*0.999;

Best Regard

**Comment by HiroIshida** [comment]
@smshariatzadeh 
It was more than two years ago when I wrote the program. As I remember, without relaxing the constraint, sometimes an error occurred. But probably just because I was immature programmer/engineer at that time and made some mistake around other parts of the code. 

As you points out, reasonably, there is no need to relax the constraint nor iterating it. So I removed it (see https://github.com/HiroIshida/robust-tube-mpc/commit/427a181dd368f0b60b1ecfa81e33e062ff0359e0).  After removing the iterating part, and I run the example program, and confirmed that it works well. Thanks.
